commerce_coordinator/apps/commercetools/views.py

```diff
@@ -39,6 +41,9 @@
 class OrderFulfillView(APIView):
     """Order Fulfillment View"""
 
+    authentication_classes = [JwtBearerAuthentication, SessionAuthentication]
```

Comment:
nit: Just setting the `JwtBearerAuthentication` class is required here.
```suggestion
    authentication_classes = (JwtBearerAuthentication,)
```

Reply:
I tried using only `JwtBearerAuthentication`, but `SessionAuthentication` is required for tests to pass.